services/terrain/src/terrain/services/filesystem/manifest.clj

```diff
@@ -1,75 +0,0 @@
-(ns terrain.services.filesystem.manifest
-  (:use [terrain.services.filesystem.common-paths]
-        [terrain.services.filesystem.validators]
-        [terrain.services.filesystem.sharing :only [anon-file-url anon-readable?]]
-        [clj-jargon.init :only [with-jargon]]
-        [clj-jargon.metadata :only [get-attribute attribute?]]
-        [slingshot.slingshot :only [try+ throw+]])
-  (:require [clojure.tools.logging :as log]
-            [clojure-commons.file-utils :as ft]
-            [dire.core :refer [with-pre-hook! with-post-hook!]]
-            [terrain.services.filesystem.validators :as validators]
-            [terrain.services.filesystem.stat :refer [detect-content-type]]
-            [terrain.services.filesystem.garnish.irods :as filetypes]
-            [tree-urls-client.core :as tree]
-            [terrain.util.config :as cfg]
-            [terrain.services.filesystem.icat :as icat])
-  (:import [org.apache.tika Tika]))
-
-(def ^:private coge-attr "ipc-coge-link")
-
-(defn- extract-tree-urls
-  [cm fpath]
-  (if (attribute? cm fpath (cfg/tree-urls-attr))
-    (-> (get-attribute cm fpath (cfg/tree-urls-attr))
-      first
-      :value
-      ft/basename
-      tree/get-tree-urls
-      :tree-urls)
-    []))
-
-(defn- extract-coge-view
-  [cm fpath]
-  (if (attribute? cm fpath coge-attr)
-    (mapv (fn [{url :value} idx] {:label (str "gene_" idx) :url url})
-          (get-attribute cm fpath coge-attr) (range))
-    []))
-
-(defn- format-anon-files-url
-  [fpath]
-  {:label "anonymous" :url (anon-file-url fpath)})
-
-(defn- extract-urls
-  [cm fpath]
-  (let [urls (concat (extract-tree-urls cm fpath) (extract-coge-view cm fpath))]
-    (vec (if (anon-readable? cm fpath)
-           (conj urls (format-anon-files-url fpath))
-           urls))))
-
-(defn- manifest-map
-  [cm user path]
-  {:action       "manifest"
```

Comment:
You'll notice this key is not in the new version. It also doesn't appear on http://cyverse.github.io/DE/api/endpoints/filesystem/manifest.html#file-manifest and we don't use it, and it wasn't useful, so good riddance.